app/http.php

```diff
@@ -0,0 +1,1039 @@
+        $interval = (int) App::getEnv('OPEN_RUNTIMES_EXECUTOR_MAINTENANCE_INTERVAL', '3600'); // In seconds
+        Timer::tick($interval * 1000, function () use ($orchestrationPool, $activeRuntimes) {
+            Console::info("Running maintenance task ...");
+            foreach ($activeRuntimes as $activeRuntimeId => $runtime) {
+                $inactiveThreshold = \time() - App::getEnv('OPEN_RUNTIMES_EXECUTOR_INACTIVE_TRESHOLD', 60);
+                if ($runtime['updated'] < $inactiveThreshold) {
+                    go(function () use ($activeRuntimeId, $runtime, $orchestrationPool, $activeRuntimes) {
+                        try {
+                            $connection = $orchestrationPool->pop();
+                            $orchestration = $connection->getResource();
+                            $orchestration->remove($runtime['name'], true);
+                            $activeRuntimes->del($activeRuntimeId);
+                            Console::success("Successfully removed {$runtime['name']}");
+                        } catch (\Throwable $th) {
+                            Console::error('Inactive Runtime deletion failed: ' . $th->getMessage());
+                        } finally {
+                            isset($connection) && $orchestrationPool->push($connection);
+                        }
+                    });
+                }
+            }
+            Console::success("Done.");
+        });
+
+        Console::success('Done.');
+
+        $server = new Server('0.0.0.0', 80, false);
+
+        $server->handle('/', function (SwooleRequest $swooleRequest, SwooleResponse $swooleResponse) {
+            $request = new Request($swooleRequest);
+            $response = new Response($swooleResponse);
+
+            $app = new App('UTC');
+
+            try {
+                $app->run($request, $response);
+            } catch (\Throwable $th) {
+                $code = 500;
+
+                /**
+                 * @var Logger $logger
+                 */
+                $logger = $app->getResource('logger');
+                logError($th, "serverError", $logger);
+                $swooleResponse->setStatusCode($code);
+                $output = [
+                    'message' => 'Error: ' . $th->getMessage(),
+                    'code' => $code,
+                    'file' => $th->getFile(),
+                    'line' => $th->getLine(),
+                    'trace' => $th->getTrace()
+                ];
+                $swooleResponse->end(\json_encode($output));
+            }
+        });
+
+        Console::success('Executor is ready.');
+
+        Swoole\Process::signal(SIGINT, fn () => removeAllRuntimes($orchestrationPool));
+        Swoole\Process::signal(SIGQUIT, fn () => removeAllRuntimes($orchestrationPool));
+        Swoole\Process::signal(SIGKILL, fn () => removeAllRuntimes($orchestrationPool));
+        Swoole\Process::signal(SIGTERM, fn () => removeAllRuntimes($orchestrationPool));
```

Comment:
Why dont we `use Swoole\Process` and then use `Process::signal()` ?

Reply:
No reason. Adding this change.